services/movie.py

```diff
@@ -0,0 +1,42 @@
+from typing import Optional, List
+
+from db.models import Movie, Genre, Actor
+
+
+def get_movies(genres_ids: Optional[List[int]] = None,
+               actors_ids: Optional[List[int]] = None) -> None:
+    movies = Movie.objects.all()
+
+    if genres_ids:
+        movies = movies.filter(genres__id__in=genres_ids)
+
+    if actors_ids:
+        movies = movies.filter(actors__id__in=actors_ids)
+
+    return movies
+
+
+def get_movie_by_id(movie_id: int) -> Movie:
+    return Movie.objects.filter(id=movie_id).first()
```

Comment:
Using `filter().first()` can return `None` if no movie is found with the given ID. The return type hint for `get_movie_by_id` should be `Optional[Movie]` to indicate that `None` can be returned.